ibek/ioc.py

```diff
@@ -171,6 +171,7 @@ class IOC:
     ioc_name: A[str, desc("Name of IOC")]
     description: A[str, desc("Description of what the IOC does")]
     entities: A[Sequence[Entity], desc("List of classes of Entity this IOC supports")]
+    generic_image: A[str, desc("The generic IOC container image registry URL")]
```

Comment:
Do we like the name generic? Would `ioc_image` or `container_name` be better?

Reply:
it is a container image not a container and it is a generic ioc image, not a ioc instance. There is no such thing as a ioc instance image so maybe shortening to ioc_image is OK. 

But I will go with generic_ioc_image for now. Feel free to override me.

Reply:
I've changed my mind already.

This class is all about generic IOC and if I use generic_ioc_image I should also use generic_ioc_name.

Instead how about changing the classname to GenericIOC ?

Reply:
hmmm. Nope. IOC derived classes are Generic and IOC dervided instances are ioc instances so IOC is better. :-)

Reply:
I rewrote the docstring and desc fields and named the offending member generic_ioc_image

```
@dataclass
class IOC:
    """
    Used to load an IOC instance entities yaml file into memory using 
    IOC.deserialize(YAML().load(ioc_instance_yaml).

    Before loading the entities file all Entity classes that it contains
    must be defined in modules.py. This is achieved by deserializing all
    support module definitions files used by this IOC and calling 
    make_entity_classes().
    """

    ioc_name: A[str, desc("Name of IOC instance")]
    description: A[str, desc("Description of what the IOC does")]
    entities: A[Sequence[Entity], desc("List of entities this IOC instantiates")]
    generic_ioc_image: A[str, desc("The generic IOC container image registry URL")]
```